korgw/src/commonMain/kotlin/korlibs/graphics/AGObjects.kt

```diff
@@ -39,6 +41,7 @@ class AGBuffer : AGObject() {
     // @TODO: This will allow to upload chunks of uniform buffers for example.
     // glBufferData & glBufferSubData
     fun upload(data: ByteArray, offset: Int = 0, length: Int = data.size - offset): AGBuffer = upload(Int8Buffer(data, offset, length).buffer)
+    fun upload(data: UByteArray): AGBuffer = upload(Uint8Buffer(data).buffer)
```

Comment:
Missing the `offset` and `length` to be consistent with the other signatures. For simplicity, since UByteArray is a wrapper of ByteArray, maybe you can do something like `upload(data.asByteArray(), offset, length)` and reuse the ByteArray method

Reply:
Not sure to keep this implementation, but if i do, il will update the code.